app/src/main/java/com/eventyay/organizer/core/organizer/detail/OrganizerDetailViewModel.java

```diff
@@ -109,7 +108,6 @@ public void uploadImage(ImageData imageData) {
                 .doOnSubscribe(disposable -> progress.setValue(true))
                 .doFinally(() -> progress.setValue(false))
                 .subscribe(uploadedImage -> {
-                    success.setValue("Image Uploaded Successfully");
```

Comment:
This should remain as it is a response to user action

Reply:
Done.